README.md

```diff
@@ -1,6 +1,11 @@
 <a href="https://leanpub.com/mobile-security-testing-guide"><img width=180px align="right" style="float: right;" src="Document/Images/mstg-cover-release-small.jpg"></a>
 
+
 # OWASP Mobile Security Testing Guide
+[![Creative Commons License](https://i.creativecommons.org/l/by-sa/3.0/88x31.png)](http://creativecommons.org/licenses/by-sa/3.0/)
```

Comment:
Add the URL titles

Reply:
This is odd, i already have that. Changed "CC" in to "CC-by-SA 3"